PhotDM.tex

```diff
@@ -1565,9 +1571,16 @@ \subsection{Interrelation between Pogson and Asinh magnitudes}
 when b=0, it is recommended to use different implementation conversion codes for both types of magnitudes as a general implementation both for Pogson (b=0) and asinh (b>0) could easily produce numerical infinites during the evaluation.
 \par
 
+\section{Data Model VODML documentation}
 
-\section{Data Model Summary}
+The VODML description of this model is available at \\ \url{http://ivoa.net/xml/}.
+
+The documentation page generated from this reference XML document is available on the Phot DM v1.1 RFC discussion page : \\
+\url{https://wiki.ivoa.net/internal/IVOA/PhotDM11RFC/Photv1_1_PR_20220520.html}
```

Comment:
If you link into the Wiki (which I think is a bad idea), at least say that's informational rather than normative.

But then: Is there a case for including these files with the document?  What tooling is necessary to build them?  I could probably be convinced to add some support for this kind of thing to ivoatex (but: perhaps we ought to chat in the TCG on the question of autogenerated docs in RECs)?